### DISCUSSION TITLE
Action is not producing any data

### DISCUSSION THREAD

**Discussion by BathiniRavi** [StackStorm/st2#5448] [General]
Triggering a command via chatbot but for some reason it is not displaying the output.

![image](https://user-images.githubusercontent.com/55383317/142220537-fc083929-c43a-4ed7-b6e9-de59a58de629.png)

Any clue what could be wrong ?

**Comment by minsis** [comment]
I would say because the action is failing before its even executed remotely given that it ran for less than a second; even a simple remote cmd like yours would take a 5-10 seconds at the minimum. You can get better output from the CLI tool `st2 execution get <id>`. Also you can grep for the execution ID inside the st2actionrunner.log. If there's nothing there then try turning on debugging and try again.
